### DISCUSSION TITLE
Error after each compilling

### DISCUSSION THREAD

**Discussion by Gouls1** [reach-sh/reach-lang#1099] [Q&A]
Hi,

After every compile, once I try to compile again, the compile comes up with either of those error message:

reach: /app/src/package.json: openFile: does not exist (No such file or directory)
or
reach: /app/src/Dockerfile: openFile: does not exist (No such file or directory)

The only way to get the compiler to compile again is to use ./reach docker-reset, then I can compile another time before the error reoccurs.

Does anyone know how to solve this issue?

Many thanks

**Comment by jeapostrophe** [comment]
@mattaudesse Can you reply, please?

**Reply by mattaudesse** [reply]
@Gouls1 what's the output if you run `ls -alh` from the same directory?

**Reply by Gouls1** [reply]
@mattaudesse So today I tried 'run' a few times in a row, and I did not get the errors anymore (I haven't changed anything though).

I still ran the command your suggested and the output is (I replaced my name with 'xxxxx') :

drwxr-xr-x  8 xxxxxxxxxxxxx  staff   256B  7 Jun 09:45 .
drwxr-xr-x  4 xxxxxxxxxxxxx staff   128B  4 Jun 18:11 ..
-rw-r--r--@ 1 xxxxxxxxxxxxx  staff   6.0K  6 Jun 10:01 .DS_Store
drwxr-xr-x  3 xxxxxxxxxxxxx staff    96B  4 Jun 21:14 .reach
drwxr-xr-x  3 xxxxxxxxxxxxx  staff    96B  4 Jun 20:43 build
-rw-r--r--@ 1 xxxxxxxxxxxxx  staff   1.8K  5 Jun 22:10 index.mjs
-rw-r--r--@ 1 xxxxxxxxxxxxx  staff   2.4K  5 Jun 22:10 index.rsh
-rwxr-xr-x  1 xxxxxxxxxxxxx  staff   9.0K  4 Jun 20:20 reach

**Reply by mattaudesse** [reply]
Thanks @Gouls1 - if the problem comes back please let us know, but I'm glad it's working for you now.

There must've been something else going on before and it'd help to gather more clues if it happens again.
